config-templates/content_indices.properties

```diff
@@ -1,4 +1,4 @@
 #
 #Mon Jan 29 14:17:14 BST 2023
-latest=05bbbb74fd52cd0aea3a048a3c1e6c434ce5efaf
-live=05bbbb74fd52cd0aea3a048a3c1e6c434ce5efaf
+latest=4bde63432762588c3e83ecdbd74485dfe23b2885
+live=4bde63432762588c3e83ecdbd74485dfe23b2885
```

Comment:
I don't think a content SHA update is required for this ticket?

Reply:
This is not for the ticket, just to have updated events and such things. Having this wouldn't hurt for the future.
If you still feel like we dont want new SHA then i'll revert it.

Reply:
We usually don't update it in a ticket unless it's required (such as adding new email template required by the code for example)

Reply:
reverted back